test/gtest/ucp/test_ucp_proto_mock.cc

```diff
@@ -219,20 +157,22 @@ class test_ucp_proto_mock : public ucp_test, public mock_component_test {
 
     virtual void init() override
     {
+        /* Reset topo provider to force reload from config */
+        ucs_sys_topo_reset_provider();
+        modify_config("TOPO_PRIO", "default");
```

Comment:
can u document each env var why is it needed?

Reply:
sure, done